metadata-models/src/main/pegasus/com/linkedin/metadata/entity/DataJobEntity.pdl

```diff
@@ -0,0 +1,27 @@
+namespace com.linkedin.metadata.entity
+
+import com.linkedin.common.DataJobUrn
+import com.linkedin.common.DataFlowUrn
+
+
+/**
+ * Data model for a DataJob entity
+ */
+record DataJobEntity includes BaseEntity {
+
+  /**
+   * Urn for the DataJob
+   */
+  urn: DataJobUrn
+
+  /**
+   * Urn of the associated DataFlow
+   */
+  flow: optional DataFlowUrn
```

Comment:
could a datajob exist without a dataflow?

Reply:
I guess in that case it should be a DataProcess 😄  I'll fix this!

Reply:
This is actually a curious case. All other entities seem to have all fields other than the urn as optional, and actually when setting this as compulsory I get:
```
> Task :metadata-models:test FAILED

Gradle suite > Gradle test > com.linkedin.metadata.ModelValidation.validateEntities FAILED
    com.linkedin.metadata.validator.InvalidSchemaException: Entity 'com.linkedin.metadata.entity.DataJobEntity' must contain an optional 'flow' field
        at com.linkedin.metadata.validator.ValidationUtils.invalidSchema(ValidationUtils.java:44)
        at com.linkedin.metadata.validator.EntityValidator.lambda$validateEntitySchema$1(EntityValidator.java:56)
        at java.util.ArrayList.forEach(ArrayList.java:1257)
        at com.linkedin.metadata.validator.EntityValidator.validateEntitySchema(EntityValidator.java:55)
        at com.linkedin.metadata.validator.EntityValidator.validateEntitySchema(EntityValidator.java:68)
        at java.util.ArrayList.forEach(ArrayList.java:1257)
        at com.linkedin.metadata.ModelValidation.validateEntities(ModelValidation.java:32)
```
Don't know the rational about that validation requirement, though 🤷‍♂️